hutte.yml

```
@@ -8,19 +8,61 @@ push_script: |
 
 custom_scripts:
   # This scripts will be displayed on the scratch org's page
-  scratch_org:
-    'Import Data':
-      description: "Import data using SFDMU"
+  scratch_org: &custom_scripts
+    "Commit Data":
+      description: "Commit data into Git using SFDMU"
       run: |
-        set -euo pipefail # fail fast
+        set -e
+        cp data/template-export.json "$HUTTE_INPUT_destinationfolder/export.json"
+        # If it contains "LIMIT", use the query as is, otherwise add LIMIT 10 to avoid accidental commit of several data records
+        if [[ "$HUTTE_INPUT_soql" =~ [Ll][Ii][Mm][Ii][Tt] ]]; then
+            soql="$HUTTE_INPUT_soql"
+        else
+            soql="$HUTTE_INPUT_soql LIMIT 10"
+        fi
+        escaped_soql=$(printf '%s\n' "$soql" | sed 's/[&/\]/\\&/g')
+        sed -i "s|<insertQuery>|$escaped_soql|g" "$HUTTE_INPUT_destinationfolder/export.json"
```

We can use `envsubst` to replace environment variables from a template file (see my suggestion to `data/template-export.json`) and write the output to the actual file:

```suggestion
        export ESCAPED_SOQL_QUERY="$(printf '%s\n' "$soql" | sed 's/[&/\]/\\&/g')"
        envsubst < data/template-export.json > "$HUTTE_INPUT_destinationfolder/export.json"
```